independent-publisher-connectors/The Meal DB/Readme.md

```diff
@@ -1,37 +0,0 @@
-# TheMealDB
```

Comment:
@MICHAEL-MAURICE can you please update your branch and remove this connector from your pull request?